doc/rados/configuration/ceph-conf.rst

```diff
@@ -4,17 +4,16 @@
  Configuring Ceph
 ==================
 
-When Ceph services start, the initialization process activates a set of
-daemons that run in the background. A :term:`Ceph Storage Cluster` runs at
+Every :term:`Ceph Storage Cluster` runs at
 least three types of daemons:
 
 - :term:`Ceph Monitor` (``ceph-mon``)
 - :term:`Ceph Manager` (``ceph-mgr``)
 - :term:`Ceph OSD Daemon` (``ceph-osd``)
 
-Any Ceph Storage Cluster that supports the :term:`Ceph File System` also runs
-at least one :term:`Ceph Metadata Server` (``ceph-mds``). Any Cluster that
-supports :term:`Ceph Object Storage` runs Ceph RADOS Gateway daemons
+A Ceph Storage Cluster that deployes the :term:`Ceph File System` also runs
```

Comment:
s/deployes/deploys/

This is a note to self, and I will fix this in a push later.

Reply:
I already did.  Less work this way.